frontend/src/index.html

```diff
@@ -18,7 +18,7 @@
 
 <body>
     <div id="app"></div>
-    <script src="http://{vcs_js_server}/scripts/vcs.js" type="text/javascript"></script>
+    <script src="http://{vcs_js_server}/vcs.js" type="text/javascript"></script>
```

Comment:
with these changes does the /scripts/autorun.sh script still work when deploying a devel server?